src/committee/orchestrator.rs

```diff
@@ -75,79 +83,120 @@ impl Orchestrator {
         // Round 1
         //
 
-        let mut commitments_map = BTreeMap::new();
+        // Round 1 - Randomly pick some members
+        let results = Arc::new(Mutex::new(vec![]));
+        let mut all_members = self.committee_cfg.members.iter().collect_vec();
```

Comment:
* "randomly pick some members" is not correct here, it's more "shuffle order of members" (the idea is that we don't contact the same ones first)
* that `iter().collect_vec()` can just be replaced by a `clone()`